frontend/packages/modelServing/src/extension-points/index.ts

```diff
@@ -1,11 +1,43 @@
-import { Extension } from '@openshift/dynamic-plugin-sdk';
+import type { ProjectKind } from '@odh-dashboard/internal/k8sTypes';
+import { CodeRef, Extension } from '@openshift/dynamic-plugin-sdk';
 
-export type ModelServingPlatform = Extension<
+export type ModelServingPlatformExtension = Extension<
   'model-serving.platform',
   {
     id: string;
     name: string;
+    manage: CodeRef<{
+      isInstalled: () => Promise<boolean>;
+      enable: (project: ProjectKind) => Promise<string>;
+      disable: (project: ProjectKind) => Promise<string>;
+      isEnabled: (project: ProjectKind) => boolean;
+    }>;
+    enableCardText: {
+      title: string;
+      description: string;
+      selectText: string;
+      enabledText: string;
+    };
+    deployedModelsView: {
+      startHintTitle: string;
+      startHintDescription: string;
+      deployButtonText: string;
+    };
```

Comment:
This seems unruly -- Curious why not separate extensions -- the overlapping "id" need?